.github/CODEOWNERS

```diff
@@ -16,6 +16,11 @@
 /packages/create-next-app/                    @vercel/next-js
 /pnpm-lock.yaml                               @vercel/next-js @vercel/turbopack
 
+# Next.js CLI
+
+/packages/next/src/bin                           @samcx @ijjk
+/packages/next/src/cli                           @samcx @ijjk
+
```

Comment:
Wanna move this to a separate PR? Also maybe keep the same three that we have on all other specific owners: (`@timneutkens @ijjk @shuding`)

Reply:
<picture data-single-emoji=":done:" title=":done:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/done/556da3823bdfab37.png" alt=":done:" align="absmiddle"></picture> , will create in separate <picture data-single-emoji=":pr:" title=":pr:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/pr/876af7e7d65e1595.png" alt=":pr:" align="absmiddle"></picture>

Reply:
https://github.com/vercel/next.js/pull/62749